packages/picasso.js/src/core/chart-components/labels/strategies/bars.js

```diff
@@ -17,6 +24,11 @@ function cbContext(node, chart) {
   };
 }
 
+function isValidText(text) {
+  const type = typeof text;
+  return (type === 'string' || type === 'number') && text !== '';
```

Comment:
I think only `type === 'string'` should be allowed

Reply:
I think a common case is where the user provide some data value that is a number. Ex: `({ data }) => data.someProp.value`.

Reply:
The return value should be coerced then (line 358)